libs/designer/src/lib/ui/panel/nodeDetailsPanel/tabs/scratchTab.tsx

```diff
@@ -106,9 +109,30 @@ export const ScratchTab = () => {
     setNaturalLanguageQueryInput(example);
   }, []);
 
+  const azureCallback = (prompt?: string) => {
+    alert(prompt);
```

Comment:
is the alert expected?

Reply:
This is just the scratch tab, the callback is normally passed back by the azure portal, but because we don't have the function in standalone, I just used an alert temporarily